config/jobs/kubernetes/publishing-bot/publishing-bot-presubmits.yaml

```diff
@@ -31,3 +32,120 @@ presubmits:
+          path_alias: k8s.io/api
+        - org: kubernetes
+          repo: apiextensions-apiserver
+          base_ref: master
+          path_alias: k8s.io/apiextensions-apiserver
+        - org: kubernetes
+          repo: apimachinery
+          base_ref: master
+          path_alias: k8s.io/apimachinery
+        - org: kubernetes
+          repo: apiserver
+          base_ref: master
+          path_alias: k8s.io/apiserver
+        - org: kubernetes
+          repo: client-go
+          base_ref: master
+          path_alias: k8s.io/client-go
+        - org: kubernetes
+          repo: cli-runtime
+          base_ref: master
+          path_alias: k8s.io/cli-runtime
+        - org: kubernetes
+          repo: cloud-provider
+          base_ref: master
+          path_alias: k8s.io/cloud-provider
+        - org: kubernetes
+          repo: cluster-bootstrap
+          base_ref: master
+          path_alias: k8s.io/cluster-bootstrap
+        - org: kubernetes
+          repo: code-generator
+          base_ref: master
+          path_alias: k8s.io/code-generator
+        - org: kubernetes
+          repo: component-base
+          base_ref: master
+          path_alias: k8s.io/component-base
+        - org: kubernetes
+          repo: csi-api
+          base_ref: master
+          path_alias: k8s.io/csi-api
+        - org: kubernetes
+          repo: csi-translation-lib
+          base_ref: master
+          path_alias: k8s.io/csi-translation-lib
+        - org: kubernetes
+          repo: kube-aggregator
+          base_ref: master
+          path_alias: k8s.io/kube-aggregator
+        - org: kubernetes
+          repo: kube-controller-manager
+          base_ref: master
+          path_alias: k8s.io/kube-controller-manager
+        - org: kubernetes
+          repo: kubelet
+          base_ref: master
+          path_alias: k8s.io/kubelet
+        - org: kubernetes
+          repo: kube-proxy
+          base_ref: master
+          path_alias: k8s.io/kube-proxy
+        - org: kubernetes
+          repo: kube-scheduler
+          base_ref: master
+          path_alias: k8s.io/kube-scheduler
+        - org: kubernetes
+          repo: metrics
+          base_ref: master
+          path_alias: k8s.io/metrics
+        - org: kubernetes
+          repo: node-api
+          base_ref: master
+          path_alias: k8s.io/node-api
+        - org: kubernetes
+          repo: sample-apiserver
+          base_ref: master
+          path_alias: k8s.io/sample-apiserver
+        - org: kubernetes
+          repo: sample-cli-plugin
+          base_ref: master
+          path_alias: k8s.io/sample-cli-plugin
+        - org: kubernetes
+          repo: sample-controller
+          base_ref: master
+          path_alias: k8s.io/sample-controller
+      spec:
+        containers:
+          - image: gcr.io/k8s-testimages/kubekins-e2e:v20190301-76bc03340-master
+            command:
+            - make
+            args:
+            - run
```

Comment:
@sttts @dims while using `make run`, I had to create the namespace `k8s-publishing-bot` for the script to work. What was I missing?

Reply:
yes, that's true. i had to create the namespace by hand as well

Reply:
Are you expecting to create namespaces in the test cluster?

Reply:
> Are you expecting to create namespaces in the test cluster?

(my bad, should have added a comment about this on Friday)

No...given the other things that this cluster will need (ref @dims' comment here https://github.com/kubernetes/test-infra/pull/11697#discussion_r263776911), can we have another cluster for this job - just like we use `test-infra-trusted` for some jobs? Can wg-k8s-infra help here?

Reply:
@dims [mentioned on slack](https://kubernetes.slack.com/archives/CCK68P2Q2/p1552315195087700) that we could also run the binary directly...I'll look into this after fixing a flake with the publishing-bot. Please don't review yet :see_no_evil: